react/ShippingForm.tsx

```diff
@@ -101,13 +105,31 @@ const ShippingForm: React.FC = () => {
 
     case matches('selectDeliveryOption'): {
       return (
-        <ShippingOptionList
-          deliveryOptions={state.context.deliveryOptions}
-          onEditAddress={() => send('EDIT_ADDRESS')}
-          onEditReceiverInfo={() => send('EDIT_RECEIVER_INFO')}
-          onDeliveryOptionSelected={handleDeliveryOptionSelect}
-          selectedAddress={state.context.selectedAddress!}
-        />
+        <>
+          {selectedAddress?.receiverName && (
+            <div className="c-muted-1">
+              <span className="fw6 flex items-center">
+                <FormattedMessage id="store/checkout.shipping.receiverLabel" />{' '}
+                <div className="dib ml4">
+                  <ButtonPlain onClick={() => send('EDIT_RECEIVER_INFO')}>
+                    <IconEdit solid />
+                  </ButtonPlain>
+                </div>
+              </span>
+
+              <div className="mt2 mb6 lh-copy">
+                {selectedAddress.receiverName}
+              </div>
+            </div>
+          )}
```

Comment:
question: how to be sure that this will never be triggered on shipping calculator?

Reply:
it seems that shipping won't use the state machine, is that right, @lucasecdb?

Reply:
it won't for now, but we'd like to make it use it at some point in the future (at least a subset of the states)